### ISSUE TITLE
Different accuracy between NCHW-NHWC data format when finetuning from imagenet

### ISSUE THREAD

**Issue by chrisrn** [tensorflow/models#3418]
### System information
- **What is the top-level directory of the model you are using**: tensorflow/models
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.4.1
- **Bazel version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: 8.0
- **GPU model and memory**: GTX 1080i
- **Exact command to reproduce**: N/A

### Describe the problem
I trained resnet_v2_50 on my data set using the pre-trained model from imageNet. I made some small modifications inside the network to accept NCHW data format. When I train using NHWC order the training accuracy is lower than the NCHW model. So my question is: Are the imageNet models trained with NCHW format? If that's true then it's reasonable to have higher accuracy using NCHW, because I restore the checkpoint correctly. According to this repo, the models are using NHWC. I also opened the checkpoints using `tensorflow/python/tools/inspect_checkpoint.py` and I see that the kernels have the same sizes even using NHWC or NCHW so I can't infer something from the checkpoints.

**Comment by asimshankar** [comment]
See https://www.tensorflow.org/performance/performance_guide#data_formats for some discussion on data formats. Generally, we use NCHW when training on NVIDIA GPUs with cuDNN.  The training script selects the format automatically (see [flags](https://github.com/tensorflow/models/blob/master/official/resnet/resnet.py#L606) and [code](https://github.com/tensorflow/models/blob/master/official/resnet/resnet.py#L314)) depending on whether or not a GPU is present.

If by "Are the imagenet models trained with NCHW format" you mean if the pre-trained models referred to in https://github.com/tensorflow/models/tree/master/official/resnet#pre-trained-model are trained in NCHW format, then yup, they are.

Closing this issue out since I believe your question is answered (and I'll send a PR to make this clearer in the README). Feel free to reopen if I'm mistaken. Thanks.

**Comment by chrisrn** [comment]
Thank you my question is answered but you have to make clear in slim documentation that pre-trained models are in NCHW since slim models are not optimized for this kind of format.

**Comment by chrisrn** [comment]
Well the network that I used was not official/resnet but the one from research/slim/nets. The functionality for training these networks with NCHW is not yet integrated but when I did it by myself I had higher accuracy
